ui/components/component-library/avatar-base/avatar-base.types.ts

```diff
@@ -20,7 +21,7 @@ export interface AvatarBaseProps extends TextProps {
    * 'AvatarBaseSize.Md'(32px), 'AvatarBaseSize.Lg'(40px), 'AvatarBaseSize.Xl'(48px)
    * Defaults to AvatarBaseSize.Md
    */
-  size?: AvatarBaseSize;
+  size?: AvatarBaseSize | AvatarNetworkSize;
```

Comment:
Not sure about this because if we carry on this pattern we will just end up with many variant types that AvatarBase accepts. There must be a TypeScript pattern that uses an argument type? I'm willing to proceed until we develop that pattern